Add how to create an API key into the README (#218)
autonomys/auto-sdk: packages/auto-drive/README.md

### How to use it?

> For interacting with auto-drive API would be needed to create an API. For that purpose, follow the following steps:

Reviewer: ```suggestion
To interact with the Auto-Drive API, you'll need to create an API key. Follow these steps:
```

- Go to [Auto-Drive](https://ai3.storage) and login with your preffered SSO.

> - Once you're logged in, click on the profile section at the left-sided menu.

Reviewer: ```suggestion
- Once you're logged in, click on the profile section in the left sidebar menu.
```

> - On the profile section, click on 'Create API Key'

Reviewer: ```suggestion
- In the profile section, click on 'Create API Key'
```
